src/js/tracks/text-track-display.js

```diff
@@ -103,7 +121,20 @@ class TextTrackDisplay extends Component {
       cues.push(track['activeCues'][i]);
     }
 
-    window['WebVTT']['processCues'](window, track['activeCues'], this.el_);
+    if (track['kind'] === 'descriptions') {
+
+      if ((overrides.descriptionsPlayback) && (overrides.descriptionsPlayback === 'screenReaderOnly')) {
```

Comment:
In which case, this shouldn't be a huge issue, though, we probably would want to think about whether we should allow having captions and subtitles on at the same time.